### DISCUSSION TITLE
Is there anyway to link the task in daily note and the kanban board?

### DISCUSSION THREAD

**Discussion by Ceasers** [mgmeyers/obsidian-kanban#396] [Ideas & Feature Requests]
I am trying to manage my tasks as: while I am writing some notes in the everyday's daily note page, during which I want to insert some tasks and I want these tasks to be also shown automatically in the Kanban list, is it possible to implement such function by Kanban plugin alone or with any other plugins together?

**Comment by ingvald** [comment]
(also ref https://github.com/mgmeyers/obsidian-kanban/discussions/525 )
